src/multiplex/box.py

```diff
@@ -8,17 +8,17 @@
 
 
 class BoxHolder:
-    def __init__(self, index, iterator, box_height, viewer):
+    def __init__(self, index, iterator, box_height, auto_collapse, viewer):
```

Comment:
considering this is a global config, i think it would be more appropriate to have this state on the `Viewer` instance.
more so considering my other suggestion (put the calll to `toggle_collapse` in a different location

Reply:
it's actually already part of the `Viewer` state in this PR so that should be straigtforward